src/vivarium_public_health/risks/distributions.py

```diff
@@ -141,87 +173,91 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
     # Public methods #
     ##################
 
-    def ppf(self, q):
-        if not q.empty:
-            q = clip(q)
-            weights = self.lookup_tables["ensemble_distribution_weights"](q.index)
+    def ppf(self, quantiles: pd.Series) -> pd.Series:
+        if not quantiles.empty:
+            quantiles = clip(quantiles)
+            weights = self.lookup_tables["ensemble_distribution_weights"](quantiles.index)
             parameters = {
-                name: parameter(q.index) for name, parameter in self.parameters.items()
+                name: param(quantiles.index) for name, param in self.parameters.items()
             }
-            ensemble_propensity = self.population_view.get(q.index).iloc[:, 0]
-            x = EnsembleDistribution(weights, parameters).ppf(q, ensemble_propensity)
+            ensemble_propensity = self.population_view.get(quantiles.index).iloc[:, 0]
+            x = rd.EnsembleDistribution(weights, parameters).ppf(
+                quantiles, ensemble_propensity
+            )
             x[x.isnull()] = 0
         else:
             x = pd.Series([])
         return x
 
 
-class ContinuousDistribution(Component):
+class ContinuousDistribution(RiskExposureDistribution):
     #####################
     # Lifecycle methods #
     #####################
 
-    def __init__(self, risk, mean, sd, distribution=None):
-        super().__init__()
-        self.risk = EntityString(risk)
-        self._distribution = distribution
-        self._parameters = self.get_parameters(mean, sd)
+    def __init__(self, risk: "Risk") -> None:
+        super().__init__(risk)
+        self._distribution = {
+            "normal": rd.Normal,
+            "lognormal": rd.LogNormal,
+        }[risk.distribution_type]
```

Comment:
This will KeyError if we try and pass in a bad risk type. Is that a common mistake? Will a KeyError be clear enough to fix?

Reply:
ContinuousDistribution is not fully supported anyway, as there are effectively no GBD risks that use this distribution. I think this is fine for now.